src/custom-elements/src/constants/color.ts

```diff
@@ -1,3 +1,16 @@
 export const TEXT_COLORS = ['', 'primary', 'success', 'warning', 'danger', 'muted', 'inverse'];
 
 export const SEMANTIC_COLORS = ['info', 'warning', 'danger', 'success'];
+
+export const ICON_COLORS = ['primary', 'success', 'warning', 'danger', 'light', 'muted', 'info', 'grey', 'secondary'];
+
+export type ICON_COLORS_TYPE =
+    'primary' |
+    'success' |
+    'warning' |
+    'danger' |
+    'light' |
+    'muted' |
+    'info' |
+    'grey' |
+    'secondary';
```

Comment:
@dani-cl-madrid @SergioQCL 
I tried exporting only array and converting it to Type to use it to define prop valid values.
Tried several techniques but non worked for me.
Any suggestions how can I avoid double exporting? Thanks

Reply:
@janibolkvadze 
You can use const assertions in the following way (mind the `as const` code at the end of the first line): 
```TypeScript
export const ICON_COLORS = ['primary', 'success', 'warning', 'danger', 'light', 'muted', 'info', 'grey', 'secondary'] as const;
export type IconColors = typeof ICON_COLORS[number];
```
Using `as const` in the ICON_COLORS declaration will make `typeof ICON_COLORS[number]` work as desired. 
`ICON_COLORS` is an array, so `typeof ICON_COLORS[number]` is the type of the values of the array (`string`), but as it has been defined `as const`, only the strings included in the definition will be part of the type instead of any string. [View documentation](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions)
`IconColors` if preferred over `ICON_COLORS_TYPE`.

Reply:
I have tried this technique but validation was throwing this error:
`Argument of type 'string' is not assignable to parameter of type '"primary" | "success" | "warning" | "danger" | "muted" | "info" | "light" | "grey" | "secondary"'.`
<img width="879" alt="Screenshot 2020-05-11 at 10 00 21" src="https://user-images.githubusercontent.com/25136131/81538024-66498000-936e-11ea-9216-7fa285b41a9a.png">
It works as a TYPE but breaks as an array